harness/determined/cli/agent.py

```diff
@@ -1,20 +1,17 @@
 import argparse
+import collections
+import operator
 import os
 import sys
 import typing
-from collections import OrderedDict
-from operator import attrgetter
 from typing import Any, Callable, Dict, List
 
 import determined.cli.render
 from determined import cli
-from determined.cli import render
+from determined.cli import render, errors
 from determined.cli import task as cli_task
```

Comment:
it'd be nice if some tool auto-merged these.

Reply:
it turns out google style guide requires one import per line: https://google.github.io/styleguide/pyguide.html#313-imports-formatting
i.e. 
```
from determined.cli import render
from determined.cli import errors 
```
but we have the formatter setup the opposite way.